components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -58,6 +74,161 @@ module.exports = {
         "Only emit events for the selected update types.",
       options: updateTypes,
     },
+    chatId: {
+      type: "string",
+      label: "Chat ID",
+      description: "Enter the unique identifier for the target chat or username of the target channel (in the format `@channelusername` or `@supergroupusername`).",
+    },
+    text: {
+      type: "string",
+      label: "Text",
+      description: "Enter or map the message text to send.",
+      optional: true,
+    },
+    parse_mode: {
+      type: "string",
+      label: "Parse Mode",
+      description: "Select [MarkdownV2-style](https://core.telegram.org/bots/api#markdownv2-style), [HTML-style](https://core.telegram.org/bots/api#html-style), or [Markdown-style](https://core.telegram.org/bots/api#markdown-style) of the text if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in your bot's message.",
```

Comment:
I guess we can use [toSingleLineString](https://github.com/PipedreamHQ/pipedream/blob/master/components/aws/sources/common/utils.mjs#L34) here as a utility function for log text in `description` props